posthog/hogql_queries/legacy_compatibility/filter_to_query.py

```diff
@@ -27,17 +28,72 @@
 from posthog.types import InsightQueryNode
 
 
-def entity_to_node(entity: BackendEntity) -> EventsNode | ActionsNode:
+def is_property_with_operator(property: Dict):
```

Comment:
Note that the frontend has `isPropertyFilterWithOperator`, but it works differently

Reply:
Good catch! I actually forgot we have something similar on the frontend. Will adapt the naming in the follow up PR, as this got superseded by work in https://github.com/PostHog/posthog/pull/17813.